Tuto-GUDHI-cover-complex.ipynb

```diff
@@ -0,0 +1,598 @@
+{
```

Comment:
<p>Should be:</p><pre class="ql-syntax" spellcheck="false">X = np.loadtxt('datasets/human.txt')
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GUDHI/TDA-tutorial/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>